training_record/utils.py

```diff
@@ -1,5 +1,10 @@
 '''Utility functions provided by training_record module.'''
 import re
+import logging
+
+
+dev_logger = logging.getLogger('django')  # pylint: disable=invalid-name
```

Comment:
Do not re-define loggers, they have already been defined in `infra.utils`, just import them.